packages/provider/src/json-rpc-provider.ts

```diff
@@ -439,26 +450,7 @@ export class JsonRpcProvider implements AbstractProvider {
       throw new Error('RPC Error')
     }
 
-    const {
-      chains,
-      jailed,
-      public_key,
-      service_url,
-      status,
-      tokens,
-      unstaking_time,
-    } = node
-
-    return {
-      address: await address,
```

Comment:
any idea why there was an await here? Address didn't seem to be an async func

Reply:
For some reason, it's a common pattern to get `address` as `string | Promise<string>` in this package.